lib/ldp-middleware.js

```diff
@@ -24,7 +24,7 @@ function LdpMiddleware (corsSettings) {
   router.copy('/*', acl.allow('Write'), copy)
   router.get('/*', index, acl.allow('Read'), get)
   router.post('/*', acl.allow('Append'), post)
-  router.patch('/*', acl.allow('Write'), patch)
+  router.patch('/*', acl.allow('Append'), patch)
```

Comment:
Sounds good; the code will check for Write vs Append downstream, as we discussed.